docs/subnets/how-to-run-cli-with-docker.md

```diff
@@ -0,0 +1,12 @@
+# How to Run Avalanche-CLI with Docker
+
+To run Avalanche-CLI in a docker container, you need to enable ipv6.
```

Comment:
ticket to complete this?

Reply:
I don't think there's work to do here? Users already use this.